designs/administrator/olm/List-of-operators-in-all-projects/list-of-operators.md

```diff
@@ -0,0 +1,41 @@
+
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+
+- When a user has selected the All Projects view from the context selector on the Installed Operators page, here they can view all of their Installed Operators. The goal of this design is to simplify how a user views the namespaces and associated operator(s) when they have selected to view the Installed Operators under All Projects.
+
+- When a user clicks the link in the table, “View 8…”, an anchor link takes the user to the Namespaces in the body of the Operator Details page. In this flow, we opt for a filterable experience in which the user can later filter by namespace or project under tabs such as Provided APIs or Events. (Scroll to view experience)
```

Comment:
It might be good to convey some of these operators as having an 'all' value in the namespace column, if the intent is to show that when the operator is installed to all namespaces. Alternatively the value would just read "View 38..." and it would be on the user to know that there are 38 namespaces on the cluster so this operator is available on all namespaces in the cluster. Would the value be "all"? Or "View all...", which might lose the fact that the operator is installed on all namespaces and just reads like you will view all the namespaces for that operator?

Perhaps this formatting of this column could change from "View 8..." to "8 namespaces" to better support the 'all' case? I feel like the user would expect to view what these namespaces are when clicking that link also.

Reply:
I like View all